src/features/users/components/users-overview.tsx

```diff
@@ -0,0 +1,206 @@
+    const postBody = {
+      name: formik.values?.name,
+      email: formik.values?.email,
+      gender: formik.values?.gender,
+      status: formik.values?.status,
+    } as User
+
+    dispatch(addUser(postBody))
+    setIsAdd(false);
+  }
+
+  const addNewUserView = (
+    <div>
+      <Box padding={2} paddingBottom={4}>
+        <Paper elevation={4}>
+          <Grid container spacing={2} paddingLeft={1} paddingRight={1}>
+            <Grid item xs={12}>
+              <Typography>
+                Add
+              </Typography>
+            </Grid>
+            <Grid item xs={6}>
+              <FormikInputField id={"email"} label={"Email"} formik={formik}/>
+            </Grid>
+            <Grid item xs={6}>
+              <FormikInputField id={"name"} label={"Name"} formik={formik}/>
+            </Grid>
+            <Grid item xs={12}>
+              <Divider />
+            </Grid>
+            <Grid item xs={6}>
+              <TextField
+                id="status"
+                name="status"
+                select
+                fullWidth
+                label="Status"
+                value={formik.values.status}
+                onChange={formik.handleChange}
+                onBlur={formik.handleBlur}
+                helperText={formik.touched.status && formik.errors.status}
+              >
+                {STATUSES.map((option: string) => (
+                  <MenuItem key={option} value={option}>
+                    {option}
+                  </MenuItem>
+                ))}
+              </TextField>
+            </Grid>
+            <Grid item xs={6}>
+              <TextField
+                id="gender"
+                name="gender"
+                select
+                fullWidth
+                label="Gender"
+                value={formik.values.gender}
+                onChange={formik.handleChange}
+                onBlur={formik.handleBlur}
+                helperText={formik.touched.gender && formik.errors.gender}
+              >
+                {GENDERS.map((option: string) => (
+                  <MenuItem key={option} value={option}>
+                    {option}
+                  </MenuItem>
+                ))}
+              </TextField>
+            </Grid>
+          </Grid>
+        </Paper>
+      </Box>
+    </div>
+  )
+
+
+  const addUserControl = (
+    <Grid container>
+      <Grid item xs={12}>
+        <IconButton onClick={() => {setIsAdd(false)}}>
+          <CloseIcon fontSize="large" />
+        </IconButton>
+      </Grid>
+      {isAdd && (
+        <Grid item xs={12} paddingTop={3}>
+          <IconButton disabled={!isSaveButtonEnabled} onClick={() => handleSaveNewUser()}>
+            <CheckIcon fontSize="large" />
+          </IconButton>
+        </Grid>
+      )}
+    </Grid>
+  )
+
+  return (
+    <Box paddingTop={5}>
+    <div>
+        <>
+          <Box paddingBottom={5}>
+            {!isAdd ? (
+              <>
+                <Typography>
```

Comment:
Change the color of the add text and icon to white so it's easier to view on the darker background.